analytics/local.env

```diff
@@ -0,0 +1,19 @@
+############################
+# DB Environment Variables
+############################
+
+# These are used by the Postgres image to create the admin user
+POSTGRES_USER=app
+POSTGRES_PASSWORD=secret123
+
+# Set DB_HOST to localhost if accessing a non-dockerized database
+DB_HOST=grants-analytics-db
+DB_NAME=app
+DB_USER=app
+DB_PASSWORD=secret123
+DB_SSL_MODE=allow
+
+# When an error occurs with a SQL query,
+# whether or not to hide the parameters which
+# could contain sensitive information.
+HIDE_SQL_PARAMETER_LOGS=TRUE
```

Comment:
Is the plan to setup connectivity to the DB from Python in the same way the API connects (our "DB Client" setup)? I ask because this param and a few of the above are for setup of the client that connects

https://github.com/HHS/simpler-grants-gov/blob/main/api/src/adapters/db/clients/postgres_client.py#L49

Reply:
> Is the plan to setup connectivity to the DB from Python in the same way the API connects 

Yes